pages/blog/posts/bowtie-intro.md

```diff
@@ -79,13 +79,13 @@ Bowtie is written in Python and published [on PyPI](https://pypi.org/project/bow
 If you have no existing preferred setup for installing Python applications, [install `pipx` using the platform-specific instructions for your OS](https://pipx.pypa.io/stable/), then run:
 
 ```sh
-$ pipx install bowtie-json-schema
+pipx install bowtie-json-schema
 ```
 
 which should give you a working Bowtie, which you can check via:
 
 ```sh
-$ bowtie --help
+bowtie --help
```

Comment:
any specific reason we removed the $ here?? And is this related to this PR context??